Source/OCMockTests/OCMockObjectMacroTests.m

```diff
@@ -177,11 +193,24 @@ - (void)testSetsUpStubsWithStructureReturnValues
 
 - (void)testSetsUpStubReturningNilForIdReturnType
 {
-    id mock = OCMClassMock([NSString class]);
+    id mock = OCMPartialMock([NSArray arrayWithObject:@"Foo"]);
+
+    OCMExpect([mock lastObject]).andReturn(nil);
+    XCTAssertNil([mock lastObject], @"Should have returned stubbed value");
+
+    OCMExpect([mock lastObject]).andReturn(Nil);
```

Comment:
Why does the test check `Nil` for methods returning `id`?

Reply:
I think I added it as an extra verification that Nil and nil were equivalent in this case and we could handle both.

Reply:
I see. I guess it isn't too uncommen for folks to use `nil` when it should be `Nil`.